OWNERS

```diff
@@ -0,0 +1,8 @@
+maintainers:
```

Comment:
I would call the file MAINTAINERS, rather than OWNERS.

Reply:
Agree. IMO it's more consistent

Reply:
Done